### DISCUSSION TITLE
How to set up a specific value in 'padding_mode' parameter

### DISCUSSION THREAD

**Discussion by robsver** [Project-MONAI/MONAI#6617] [Q&A] [answered]
Hi everyone,

During my data augmentation task I was wondering how to set up a constant value in `padding_mode` parameter that I found in several transforms, for example `monai.transforms.Rotate`, as long as it's not `border, reflection or zeros`. 

Thanks for your time,
Roberto

**Comment by KumoLiu** [comment, marked as answer]
Hi @robsver, we can not support setting a constant value in `padding_mode` in `Rotate` since we are following https://pytorch.org/docs/stable/generated/torch.nn.functional.grid_sample.html for now.

Thanks!

**Reply by robsver** [reply]
Hi @KumoLiu, thanks the same for answering me. 

Have a nice day, 
Roberto
